src/main/java/org/veupathdb/eda/binaryfiles/dumper/Main.java

```diff
@@ -39,16 +39,21 @@ public static void main(String[] args) throws Exception {
     String connectionPassword = getRequiredVar("APPDB_PASS");
 
     // instantiate a connection to the database
-    try (DatabaseInstance appDb = new DatabaseInstance(SimpleDbConfig.create(
-        SupportedPlatform.ORACLE, connectionUrl, connectionUser, connectionPassword))) {
-
-      DataSource ds = appDb.getDataSource();
-      StudyFactory studyFactory = new StudyFactory(ds, APP_DB_SCHEMA, StudyOverview.StudySourceType.CURATED);
+    try (DatabaseInstance metadataScanningDbInstance = new DatabaseInstance(SimpleDbConfig.create(
```

Comment:
Replace with single DatabaseInstance where config takes "2" for connection pool size to fix earlier breakage.